### ISSUE TITLE
Ideas to discuss

### ISSUE THREAD

**Issue by teresa-m** [BackofenLab/Cherri#38]
Important:
- Which model should be our core model?
- What figure/table should be in the main text or supplement
- should we store the models/ datasets somewhere zenodo
- should we give a possible use case for Cherri. Where it can be helpful to solve a problem, close a gap?

For later:
- parralization of IntaRNA calles in pos/neg data
- is there a standard file format for RRIs we can use for Eval mode?
- a new method of hutter group to deep learning for tabular data could be used to build a model
- build in cross-validation for the model to also report an f1 score
- check memory consumption of feature selection + optimization and see if the parallel is an issue here
- homologs: https://github.com/teresa-m/Cherri/issues/32
- go away from genome-only sequences with context. e.g. give directly a mRNA
- update IntaRNA to genomic version IntaRNA

**Comment by teresa-m** [comment]
Tasks:

- [ ] Report feature importance
- [ ] get [conda pacakge](https://github.com/bioconda/bioconda-recipes/pull/35246) running
- [ ] Galaxy wrapper?
- [ ] Affiliations of @Martin and Egg
- [ ] train models for single classes e.g. miRNA-mRNA, snoRNA-rRNA
- [ ] update documentaion + paper on how to change IntaRNA paramteres
- [ ] Make the training data merge better usable
- [ ] Make a more generic input data possible (no need for Chira but it needs a header)
- [ ] Get LIGR-seq, SPLASH running. Or we take a CLIP-like method as an additional validation set. 
- [ ] Get functional testing/ GitHub actions
